Example/HorizonCalendarExample/HorizonCalendarExample/Demo View Controllers/PartialMonthVisibilityDemoViewController.swift

```diff
@@ -0,0 +1,122 @@
+// Created by Bryan Keller on 6/23/20.
+// Copyright © 2020 Airbnb Inc. All rights reserved.
+
+import HorizonCalendar
+import UIKit
+
+// MARK: - PartialMonthVisibilityDemoViewController
+
+final class PartialMonthVisibilityDemoViewController: UIViewController, DemoViewController {
+
+  // MARK: Lifecycle
+
+  init(monthsLayout: MonthsLayout) {
+    self.monthsLayout = monthsLayout
+    super.init(nibName: nil, bundle: nil)
+  }
+
+  required init?(coder: NSCoder) {
+    fatalError("init(coder:) has not been implemented")
+  }
+
+  // MARK: Internal
+
+  override func viewDidLoad() {
+    super.viewDidLoad()
+
+    title = "Partial Month Visibility for Vertical Months Layout"
+
+    if #available(iOS 13.0, *) {
+      view.backgroundColor = .systemBackground
+    } else {
+      view.backgroundColor = .white
+    }
+
+    calendarView.daySelectionHandler = { [weak self] day in
+      guard let self = self else { return }
+
+      self.selectedDay = day
+      self.calendarView.setContent(self.makeContent())
+    }
+    view.addSubview(calendarView)
+
+    calendarView.translatesAutoresizingMaskIntoConstraints = false
+    switch monthsLayout {
+    case .vertical:
+      NSLayoutConstraint.activate([
+        calendarView.topAnchor.constraint(equalTo: view.layoutMarginsGuide.topAnchor),
+        calendarView.bottomAnchor.constraint(equalTo: view.layoutMarginsGuide.bottomAnchor),
+        calendarView.leadingAnchor.constraint(
+          greaterThanOrEqualTo: view.layoutMarginsGuide.leadingAnchor),
+        calendarView.trailingAnchor.constraint(
+          lessThanOrEqualTo: view.layoutMarginsGuide.trailingAnchor),
+        calendarView.centerXAnchor.constraint(equalTo: view.centerXAnchor),
+        calendarView.widthAnchor.constraint(lessThanOrEqualToConstant: 375)
+      ])
+    case .horizontal(let monthWidth):
+      NSLayoutConstraint.activate([
```

Comment:
Maybe we could add an alert that pops up when you open this config indicating that it does nothing since this option only works for `.vertical` layouts.

Reply:
Ah, I should probably just hide it when that segmented control is on horizontal. Good call!